app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -64,12 +66,12 @@ def feeds
     end
 
     def machine_id(feed)
-      os_resource_for(feed).properties['Machine Id']
+      os_resource_for(hawk_escape_id(feed)).properties['Machine Id']
     end
 
     def os_resource_for(feed)
       with_provider_connection do |connection|
-        os = os_for(feed)
+        os = os_for(hawk_escape_id(feed))
```

Comment:
Because until now, the feed id was only uuid => no need to escaping, but if the WF is running in the domain mode, by default the feed id is 'master.Unnamed Domain' the space needs to be encoded as `%20` to work properly.

In other works, it's a bug, but it has not manifested itself because the feed id was always "nice".